src/Integration.Vsix/CFamily/PortedFromJava/Protocol.cs

```diff
@@ -177,6 +178,12 @@ public static Response Read(BinaryReader reader, string issueFilePath = null)
                   throw new InvalidDataException("Communication issue with the C/C++ analyzer");
               case "message":
                   var message = readMessage(reader);
+
+                  if ("ParsingError".Equals(message.RuleKey))
```

Comment:
What is the reason for this change? "ParsingError" is a valid issue and should be displayed to the user when enabled in addition to the other issues.

Reply:
My bad, I didn't see them and I thought that it wasn't handled. But it's due to the fact that they had no file location, so they just failed like all issues without location.
I'll revert this change.